_presentations/scipy2016.md

```diff
@@ -441,6 +457,6 @@ or
 
 1. … Rename pyglet_helper to vpython-pyglet?
 3. … 3D Text
-2. … Make vpython-jupyter compatible with pyglet_helper
+2. … Make vpython-jupyter  compatible with pyglet_helper
```

Comment:
I'm pretty sure that won't happen...the glowscript API is the one that has essentially been adopted as the "official" one, and is baked into the latest version of the *Matter and Interactions* textbook, which is the only major one I know of that uses vpython (it is wrtten by Bruce Sherwood and Ruth Chaaby). A more formal specification of an API, and a set of tests that check API implementation may be doable though.